docs/reference/filebeat/filebeat-input-gcs.md

```diff
@@ -369,6 +371,95 @@ filebeat.inputs:
     poll_interval: 11m
 ```
 
+### Custom properties [attrib-custom-properties]
+
+Some object properties can be `set` or `overridden` at the input level with the help of certain configuration options. Allowing users to set or override custom object properties provides more flexibility when reading objects from a remote storage where the user might only have read access.
```

Comment:
Do "set" and "overridden" need to be in code font? ISTM these are just words. If they are not just words, what do they refer to in the context of what the user knows (i.e. not implementation details)?